src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,85 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+
+    private LinkedList<Entry<K, V>>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this(DEFAULT_CAPACITY);
+    }
+
+    public MyHashMap(int initialCapacity) {
+        table = new LinkedList[initialCapacity];
+        size = 0;
+    }
+
+    static class Entry<K, V> {
```

Comment:
Remember to parameterize the Node class. The inner class `Entry` should be parameterized with `<K, V>` to ensure type safety.